content/social-media.md

```diff
@@ -148,18 +148,18 @@ People who use social media have different abilities. For example, they may read
 ### The tool help create accessible multimedia
 
 <p><span class="persona">Persona: </span>
-  Enhamed manages the social profile of a client. He uploads images and videos to showcase his client's activities. His client now asks him to make these media accessible, but he does not know well how to do it.
+  Enhamed manages the social profile of a client. He uploads images and videos to showcase his client's activities. His client has asked him to make these media accessible, but he does not know how to do this.
 </p>
 <div class="quotes">
   <ul>
     <li>
       <p><span class="issue">Problem: </span><span><q>
-        My client has now requested me to make all media accessible, including images, audio, and video. I am not sure how to get this done, and I can’t find anything on my tool that explains it.
+        My client has now requested that I make all media accessible, including images, audio, and video. I am not sure how to get this done, and I can’t find anything in my tool that explains it.
       </q></span></p>
     </li>
     <li>
       <p><span class="issue">Works well: </span><span><q>
-        When I upload an image, the tool asks me if it needs alternative text and gives guidance on how to write it. When I upload a video, the tool suggests me to add transcripts, captions, and descriptions as appropriate.
+        When I upload an image, the tool asks me if it needs alternative text and gives guidance on how to write it. When I upload a video, the tool suggests that I add transcripts, captions, and descriptions as appropriate.
```

Comment:
Not sure what's changed in this line.

Reply:
Changed 'suggests me to' to 'suggests that I'. Just a bit of a grammar correction